app/models/authenticator/httpd.rb

```diff
@@ -146,7 +150,7 @@ def user_details_from_headers(username, request)
         user_headers.each { |k, v| log_auth_debug("  %-24{key} = \"%{val}\"" % {:key => k, :val => v}) }
       end
 
-      groups     = CGI.unescape(user_headers['X-REMOTE-USER-GROUPS'] || '').split(/[;:,]/)
+      groups     = CGI.unescape(user_headers['X-REMOTE-USER-GROUPS'] || '').split(/[#{self.class.group_delimiters}]/)
```

Comment:
Since this can come from user input, I think we need to be more careful here.  Thinking:

```suggestion
      groups     = CGI.unescape(user_headers['X-REMOTE-USER-GROUPS'] || '').split(/[#{Regexp.escape(self.class.group_delimiters)}]/)
```

Reply:
For example, what happens if a user puts `]` (because they can)

Reply:
yeah, I'm not sure how to do it.  We probably need to worry about typos and security issues with the input.

Reply:
solution: treat the delimiter as static text.

1. We will allow multiple characters
2. We will not need special escaping